src/chunk_constraint.c

```diff
@@ -224,11 +224,11 @@ ts_chunk_constraints_add_from_tuple(ChunkConstraints *ccs, const TupleInfo *ti)
 	Name hypertable_constraint_name;
 	bool should_free;
 	HeapTuple tuple = ts_scanner_fetch_heap_tuple(ti, false, &should_free);
-	MemoryContext oldcxt;
+	MemoryContext oldctx;
```

Comment:
both spellings were present in equal numbers, this happens to be my preference and I think generally dominant...